--- nagios-snmp-plugins-1.2/check_snmp_disk.c	2008-01-27 20:40:06.000000000 +0200
+++ nagios-snmp-plugins-1.2-fmt/check_snmp_disk.c	2008-11-07 13:33:45.991161415 +0200
@@ -223,12 +223,12 @@
   {
 	if(listing)
 	{
-		printf( "Checked %d disks. ( ", cnt );
-		for(i=0; i < cnt; i++)
+		printf( "Checked %d disks (%s", cnt, diskname[0]);
+		for(i=1; i < cnt; i++)
 		{
-			printf( "%s ", diskname[i] );
+			printf( ", %s", diskname[i] );
 		}
-		printf(")\n" );
+		printf(").\n" );
 	}
 	else
     	printf("Checked %d disks.\n", cnt);
--- nagios-snmp-plugins-1.2/check_snmp_proc.c	2008-01-27 20:40:06.000000000 +0200
+++ nagios-snmp-plugins-1.2-fmt/check_snmp_proc.c	2008-11-07 13:33:55.455705814 +0200
@@ -281,12 +281,12 @@
   {
 	if(listing)
 	{
-		printf("Checked %d process groups. ( ", cnt);
-		for(i=0; i < cnt; i++)
+		printf("Checked %d process groups (%s", cnt, procname[0]);
+		for(i=1; i < cnt; i++)
 		{
-			printf( "%s ", procname[i] );
+			printf( ", %s", procname[i] );
 		}
-		printf( ")\n");
+		printf( ").\n");
 	}
 	else
     	printf("Checked %d process groups.\n", cnt);
